<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Legion-5" failures="1" tests="2" name="Test" time="19.78" errors="0" timestamp="04 Sep 2021 08:48:36 GMT">
  <testcase classname="qaautomation.august2021.AppTest" name="test" time="8.051">
    <failure type="java.lang.AssertionError" message="expected [fullstackdemo] but found [fullstackdemofailed]">
      <![CDATA[java.lang.AssertionError: expected [fullstackdemo] but found [fullstackdemofailed]
at qaautomation.august2021.AppTest.test(AppTest.java:29)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase classname="qaautomation.august2021.AppTest" name="test1" time="8.36"/>
</testsuite> <!-- Test -->
